### DISCUSSION TITLE
Feedback on Growatt MOD 12KTL3-X and a question

### DISCUSSION THREAD

**Discussion by Hemmahackaren** [johanmeijer/grott#161] [General]
I have been using Grott with my Growatt MOD 12KTL3-X in proxy mode for a few days and after specifying 'invtype=tl3' and 'time=server' everything seems to work fine. Without 'time=server' the time was not reliable.

I am using the docker version with extension grotcsv (BTW it would be nice if extensions are included in docker containers).

Finally a question I haven't been able to find the answer to elsewhere:

What is the difference between pvenergytotal (named EacTotal in exports from Growatts website) and epvtotal?

I initially thought pvenergytotal was before energy loss in inverter and epvtotal was energy delivered by inverter (epvtotal is always slightly less than pvenergytotal) but got confused after trying to find the answer to this.

**Comment by johanmeijer** [comment]
Thank you for the feedback.

It is a good idea to include extensions in the container. It is always possible with docker parameters to reference to an external file / volume but having it already available is easier. 

You already described the difference of pvenergytotal (eactotal) and epvtotal I think. epvtotal is the energy that is delivered by the solarpanels. pvenergytotal is the energy delivered to the grid.  That is epvtotal - loss of energy in the inverter.

**Comment by Hemmahackaren** [comment]
In another Growatt-related software I saw the following definition (not sure what their source of information is):

eactotal=generated grid energy total
epvtotal=generated solar energy total

That aligns with what you write. The term EAC indicates it could be Energy Alternating Current and EPV indicates Energy PhotoVoltaic which also hints that what you write is correct.

But in my case (and all samples I have found on the net) EAC is always higher than EPV - which for me hints it is the other way around. Or am I confusing things and I cannot compare Watt/kWh in and Watt/kWh out without taking into account difference in voltage etc? I admit my knowledge of electricity sadly is almost non existent...

**Comment by pimisen** [comment]
> But in my case (and all samples I have found on the net) EAC is always higher than EPV - which for me hints it is the other way around. Or am I confusing things and I cannot compare Watt/kWh in and Watt/kWh out without taking into account difference in voltage etc? I admit my knowledge of electricity sadly is almost non existent...

I pretty much have the same question and also wondering if it is not the other way round ....
